CheckDrive.Mobile/CheckDrive.Mobile/CheckDrive.Mobile/AppShell.xaml

```diff
@@ -49,9 +51,10 @@
             <!--
                 Custom Style you can apply to any Flyout Item
             -->
-            <Style ApplyToDerivedTypes="True"
-                   Class="MenuItemLayoutStyle"
-                   TargetType="Layout">
+            <Style
+                ApplyToDerivedTypes="True"
+                Class="MenuItemLayoutStyle"
+                TargetType="Layout">
```

Comment:
Update your formatter, we should all use the same formatting options, otherwise each change would be showing unnecessary changes in formatting which would be distracting the reviewer.

Reply:
style should be the same as it was before, attributes should start on the same level as the Element